### PR TITLE
PR: feature/281

### PR DESCRIPTION
**Pull request by jaswsinc**
See: websharks/comment-mail#281

### CODE REVIEW

File: src/includes/templates/type-a/email/comment-notification/message.php [hunk 1]
```diff
@@ -139,22 +149,26 @@
                 <?php endif; ?>
                 <?php echo ':'; ?>
             </p>
-            <p style="font-style:italic;">
-                <?php echo esc_html($_comment_parent_clip); ?>
-            </p>
+            <div style="margin:1em 0 1em 0; font-style:italic;">
+                <?php echo $_comment_parent_content; ?>
```
Comment (raamdev) [inline src/includes/templates/type-a/email/comment-notification/message.php]: @jaswsinc Any reason this isn't escaped with `esc_html()`?

File: src/includes/templates/type-a/email/comment-notification/message.php [hunk 2]
```diff
@@ -139,22 +149,26 @@
...
-                <?php else : // The site is not collecting comment author names. ?>
+                <?php else : // The site is not collecting comment author names.?>
                     <?php echo sprintf(__('This reply was posted %1$s.', SLUG_TD), esc_html($_comment_time_ago)); ?>
                 <?php endif; ?>
             </p>
-            <p style="font-size:130%;">
-                <?php echo esc_html($_comment_clip); ?>
-            </p>
+            <div style="margin:1em 0 1em 0; font-size:130%;">
+                <?php echo $_comment_content; ?>
```
Comment (raamdev) [inline src/includes/templates/type-a/email/comment-notification/message.php]: @jaswsinc Same here: any reason this isn't escaped with `esc_html()`?

File: src/includes/templates/type-a/email/comment-notification/message.php [hunk 3]
```diff
@@ -165,30 +179,34 @@
...
-                <?php else : // The site is not collecting comment author names. ?>
+                <?php else : // The site is not collecting comment author names.?>
                     <?php echo sprintf(__('This comment was posted %1$s.', SLUG_TD), esc_html($_comment_time_ago)); ?>
                 <?php endif; ?>
             </p>
-            <p style="font-size:130%;">
-                <?php echo esc_html($_comment_clip); ?>
-            </p>
+            <div style="margin:1em 0 1em 0; font-size:130%;">
+                <?php echo $_comment_content; ?>
```
Comment (raamdev) [inline src/includes/templates/type-a/email/comment-notification/message.php]: @jaswsinc Also here: shouldn't this be escaped with `esc_html()`?

File: src/includes/templates/type-s/email/comment-notification/message.php [hunk 4]
```diff
@@ -87,38 +87,49 @@
...
+        <?php if ($_comment_parent) : // This is a reply to someone??>
 
             <?php echo $template->snippet(
                 'message-in-response-to.php',
                 [
                     '[comment_parent_url]'         => esc_attr($_comment_parent_url),
                     '[comment_parent_id]'          => esc_html($_comment_parent->comment_ID),
                     '[comment_parent_author]'      => esc_html($_comment_parent->comment_author),
-                    '[comment_parent_clip]'        => esc_html($_comment_parent_clip),
+                    '[comment_parent_content]'     => $_comment_parent_content, '[comment_parent_clip]' => $_comment_parent_content,
```
Comment (raamdev) [inline src/includes/templates/type-s/email/comment-notification/message.php]: @jaswsinc Here too: any reason this isn't escaped with `esc_html()`?

File: src/includes/templates/type-s/email/comment-notification/message.php [hunk 5]
```diff
@@ -130,12 +141,16 @@
                     '[comment_id]'       => esc_html($_comment->comment_ID),
                     '[comment_time_ago]' => esc_html($_comment_time_ago),
                     '[comment_author]'   => esc_html($_comment->comment_author),
-                    '[comment_clip]'     => esc_html($_comment_clip),
+                    '[comment_content]'  => $_comment_content, '[comment_clip]' => $_comment_content,
```
Comment (raamdev) [inline src/includes/templates/type-s/email/comment-notification/message.php]: @jaswsinc And here too: any reason this isn't escaped with `esc_html()`?

File: src/includes/templates/type-a/email/comment-notification/message.php (hunk 1 above)
Comment (jaswsinc) [inline src/includes/templates/type-a/email/comment-notification/message.php]: Yes, this is either a clip (i.e., already escaped, plain text), or it is raw HTML that makes up the already-sanitized comment content body. So escaping it again would corrupt the HTML.

**Pull request merged**